scripts/Jenkinsfile_batch_test

```diff
@@ -0,0 +1,132 @@
+                  TEST_ARGS = "${TEST_ARGS}" + " -tags=${test_name} -installVersionOrCommit ${VERSION_OR_COMMIT}"
+                  if ( test_name == "upgrademanual") {
+                    TEST_ARGS = "${TEST_ARGS}" +  " -channel ${UPGRADE_CHANNEL} "
+                  }
+                }
+              }
+
+              if ( test_name == "deployrancher" ) {
+                if ( "${CERT_MANAGER_VERSION}" != "") {
+                  TEST_ARGS = "${TEST_ARGS}" + " -certManagerVersion ${CERT_MANAGER_VERSION}"
+                }
+                if ( "${CHARTS_VERSION}" != "") {
+                  TEST_ARGS = "${TEST_ARGS}" + " -chartsVersion ${CHARTS_VERSION}"
+                }
+                if ( "${RANCHER_VERSION}" != "") {
+                  TEST_ARGS = "${TEST_ARGS}" + " -rancherVersion ${RANCHER_VERSION}"
+                }  
+                if ( "${CHARTS_REPO_NAME}" != "") {
+                  TEST_ARGS = "${TEST_ARGS}" + " -chartsRepoName ${CHARTS_REPO_NAME}"
+                }
+                if ( "${CHARTS_REPO_URL}" != "") {
+                  TEST_ARGS = "${TEST_ARGS}" + " -chartsRepoUrl ${CHARTS_REPO_URL}"
+                }
+                if ( "${CHARTS_ARGS}" != "") {
+                  TEST_ARGS = "${TEST_ARGS}" + " -chartsArgs ${CHARTS_ARGS}"
+                }
+              }
+          
+              def params = [
+                string(name: 'TEST_ARGS', value: "${TEST_ARGS}"),
+                string(name: 'REPO', value: "${REPO}"), 
+                string(name: 'BRANCH', value: "${BRANCH}"),                                         
+                string(name: 'PRODUCT_NAME', value: "${PRODUCT_NAME}"),                    
+                string(name: 'PRE_UPGRADE_VERSION', value: "${PRE_UPGRADE_VERSION}"),
+                string(name: 'VERSION_OR_COMMIT', value: "${VERSION_OR_COMMIT}"),
+                string(name: 'INSTALL_MODE', value: "${INSTALL_MODE}"),
+                string(name: 'CHANNEL', value: "${CHANNEL}"),
+                string(name: 'UPGRADE_CHANNEL', value: "${UPGRADE_CHANNEL}"),                
+                string(name: 'HOSTNAME_PREFIX', value: "${HOSTNAME_PREFIX}"), 
+                string(name: 'TEST_DIRECTORY', value: actual_testdir_name),
+                string(name: 'AWS_SSH_KEY_NAME', value: "${AWS_SSH_KEY_NAME}"),
+                string(name: 'AWS_SSH_PEM_KEY', value: "${AWS_SSH_PEM_KEY}"),                                                          
+              ]
+              echo "Params are: ${params}"
+
+              echo "Trigger build job for: ${test_name}"
+              switch(test_name) {
+                case "validatecluster":
+                  jobs[test_name] = { build job: "${env.PRODUCT_NAME}_validate_cluster", parameters: params }                 
+                case "restartservice":
+                  jobs[test_name] = { build job: "${env.PRODUCT_NAME}_restart_service", parameters: params }                  
+                case "dualstack":
```

Comment:
Do we want to add dual stack and deploy rancher as part of this batch run? @rancher-max

Reply:
it can be here. we dont have to pass it in the TEST_DIRECTORY_LIST if we dont want to?

Reply:
yes we do want to add those please!